babel.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-react'],
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          esmodules: false,
+        },
+      },
+    ],
+  ],
+  plugins: [
+    '@babel/plugin-syntax-dynamic-import',
+    '@babel/plugin-transform-arrow-functions',
+    '@babel/plugin-transform-destructuring',
+    '@babel/plugin-transform-function-name',
+    '@babel/plugin-transform-parameters',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-transform-classes',
+    '@babel/plugin-transform-shorthand-properties',
+    '@babel/plugin-transform-regenerator',
+  ],
+  sourceType: 'unambiguous',
```

Comment:
What is this for ?

Reply:
`sourceType` consider a file as a module if it contains import and/or export